web/frontend/src/components/slam-2d-render.vue

```diff
@@ -78,6 +108,21 @@ const disposeScene = () => {
   scene.clear();
 };
 
+// Find the desired color bucket for a given probability. This assumes the probability will be a value from 0 to 100
+const probToColorMapBucket = (normProb: number, numBuckets: number): number => {
```

Comment:
Could you please add unit tests for this function that shows how it behaves with probabilities both within the expected range & outside the expected range? 

You should be able to follow a similar patter to what is done here: https://github.com/viamrobotics/rdk/blob/main/web/frontend/tests/unit/Base.spec.ts#L1

I'm happy to pair with you on it.

Reply:
No current unit testing framework is set up yet for RDK, recommend just moving forward without tests until we complete https://viam.atlassian.net/browse/APP-1600

Reply:
cool stuff, added a ticket for SLAM to make some tests after that one https://viam.atlassian.net/browse/RSDK-2606